examples/cookbook/utils/verify-signature.js

```diff
@@ -0,0 +1,37 @@
+const { connect, keyStores } = require("near-api-js");
+const path = require("path");
+const homedir = require("os").homedir();
+const nacl = require("tweetnacl");
+const sha256 = require("js-sha256");
+
+const ACCOUNT_ID = "near-example.testnet";
+const CREDENTIALS_DIR = ".near-credentials";
+
+const credentialsPath = path.join(homedir, CREDENTIALS_DIR);
+const keyStore = new keyStores.UnencryptedFileSystemKeyStore(credentialsPath);
+
+const config = {
+    keyStore,
+    networkId: "testnet",
+    nodeUrl: "https://rpc.testnet.near.org",
+};
+
+verifySignature();
+
+async function verifySignature() {
+    const near = await connect({ ...config, keyStore });
+    const account = await near.account(ACCOUNT_ID);
+    const tokenMessage = new TextEncoder().encode(`Hello world, ${ACCOUNT_ID}`);
+    const signatureData = await account.connection.signer.signMessage(
+        tokenMessage,
+        ACCOUNT_ID,
+        "testnet"
+    );
+    const hash = new Uint8Array(sha256.sha256.array(tokenMessage));
```

Comment:
Why use `new Uint8Array()` here, but use `TextEncoder.encode()` above?

Since this is an example, if there is a reason for the difference it'd be nice if there was a comment explaining why here.

However, if they are effectively doing the same thing, maybe rather than use `sha256.array()`, could we make the composition of the Uint8Array here be done consistently by using e.g. `new TextEncoder().encode(sha256.sha256(tokenMessage))` directly, since `encode()` _returns a Uint8Array for us_?

Reply:
Refactored the code based on @mehtaphysical 's suggestion :)